addons/mail/static/src/models/message/message.js

```diff
@@ -773,6 +869,16 @@ function factory(dependencies) {
             compute: '_computePrettyBody',
             dependencies: ['body'],
         }),
+        /**
+         * This value is pretty body with insertion of read more/less.
+         * The initial value is always folded read more/less content.
+         * User interaction on the read more/less links affects fold state of
+         * the read more/less content of pretty body.
+         */
+        prettyBodyWithReadMoreLess: attr({
```

Comment:
I'm confused about the format of this field, is it an html string? An actual element? A jquery element?

Reply:
It's jquery. I should clarify this in docstring.